assets/training/finetune_acft_image/components/pipeline_components/hf_classification/spec.yaml

```diff
@@ -456,30 +456,30 @@ jobs:
       pytorch_model_folder: ${{parent.outputs.pytorch_model_folder}}
   model_prediction:
     type: command
-    component: azureml:model_prediction:0.0.15
-    compute: ${{parent.inputs.compute_model_evaluation}}
+    component: azureml:model_prediction:0.0.17
+    compute: '${{parent.inputs.compute_model_evaluation}}'
```

Comment:
Why single quotes?

Reply:
Handles in case there's spaces in the input; it's a good practice to have quotes enclosed bash variables.